tests/sentry/integrations/github/test_issues.py

```diff
@@ -24,10 +25,11 @@ def request(self):
     def setUp(self):
         self.user = self.create_user()
         self.organization = self.create_organization(owner=self.user)
-        self.model = Integration.objects.create(
-            provider="github", external_id="github_external_id", name="getsentry"
-        )
-        self.model.add_organization(self.organization, self.user)
+        with assume_test_silo_mode(SiloMode.CONTROL):
+            self.model = Integration.objects.create(
```

Comment:
[`self.create_integration`](https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/factories.py#L1452) can be helpful for these too